crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -571,7 +571,7 @@ def proposal_update
           flash_and_log_exception(e)
         end
       elsif params[:submit] == t("barclamp.proposal_show.commit_proposal")
-        @proposal = Proposal.where(barclamp: params[:barclamp], name: params[:id] || params[:name]).first
+        @proposal = Proposal.find_by(barclamp: params[:barclamp], name: params[:id] || params[:name])
```

Comment:
Metrics/LineLength: Line is too long. [101/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)